src/AbstractPolygon.jl

```diff
@@ -1,6 +1,9 @@
 export AbstractPolygon,
        tohrep,
-       tovrep
+       tovrep,
+       jump2pi
```

Comment:
Do we actually have to export `jump2pi`?

Reply:
not really, let's take it out.